06-user-events/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "06-user-events",
+  "version": "1.0.0",
+  "description": "Vorstellung von React-State zum Dynamisieren einer React-Applikation",
```

Comment:
This is the old description of episode 5, I guess, can you please update this one?